packages/assurance/js/index.d.ts

```diff
@@ -10,7 +10,7 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 
 */
-export class AEPAssurance{
+export class Assurance{
```

Comment:
Let's also update docs in Assurance.js, all occurrences  AEPAssurance -> Assurance

Reply:
Good catch, the docs are updated now.